cmd/run.go

```diff
@@ -448,13 +444,16 @@ a commandline interface for interacting with it.`,
 
 		// Print the end-of-test summary.
 		if !conf.NoSummary.Bool {
+			s, err := ui.NewSummary(conf.SummaryTrendStats)
+			if err != nil {
+				// Shouldn't happen, config should be valid at this point
+				logrus.WithError(err).Error("Summary error")
+				return ExitCode{errors.New("summary error"), summaryErrorCode}
```

Comment:
I don't like this. Should we just ignore err (`s, _ := ui.NewSummary...`) here, since it technically can't happen?

I wouldn't want to remove returning error from `NewSummary()`, since it makes sense to validate it there.

Reply:
Yeah, I don't like this either. And I don't think `summaryErrorCode` should be a thing. If we have a problem with the config, it should be returned much earlier than this place, and the error code should be `invalidConfigErrorCode`.

Reply:
Resolved in b4a776dc.